tests/testqPrefGeneral.cpp

```diff
@@ -23,26 +23,16 @@ void TestQPrefGeneral::test_struct_get()
 
 	auto tst = qPrefGeneral::instance();
 
-//	prefs.auto_recalculate_thumbnails = true;
-//	prefs.default_cylinder = copy_qstring("new base11");
 	prefs.default_filename = copy_qstring("new base12");
```

Comment:
and now you remove them all. Let's just remove them where they were commented out (and explain why removing them is the right thin in the commit message, maybe)